tests/libs/sniff_mqtt.py

```
@@ -0,0 +1,219 @@
+        elif message.HasField('commandResponse'):
+            self._print('\tCommand response sent')
+        else:
+            self._print('\tMessage with unhandled type')
+        if self._verbose: self._print(message)
+
+
+    def _decode_external_server_message(self, payload: bytes):
+        message = ExternalProtocol_pb2.ExternalServer()
+        message.ParseFromString(payload)
+        if message.HasField('command'):
+            self._handle_command(getattr(message, 'command'))
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)